client/src/shared/searchbar/searchbarController.js

```diff
@@ -5,10 +5,12 @@ sharedModule.controller('SearchbarController', ['$state', function SearchbarCont
     this.input = $state.params.query || '';
     this.submitForm = ($event) => {
         $event.preventDefault();
+        if(this.input) {
```

Comment:
Please keep indentation consistent and switch to early return instead if the string is empty

Reply:
Changed!